spec/engines/cached_search_api_connection_spec.rb

```diff
@@ -35,29 +30,20 @@
     end
   end
 
-  describe '#get', vcr: { record: :skip } do
-    context 'on cache hit' do
-      before do
-        expect(cache).to receive(:read).with(endpoint, params).and_return(response)
-      end
+  describe '#get' do
+    context 'when response has not been cached' do
+      it 'calls the api' do
+        expect(cached_connection.connection).to receive(:get).with(endpoint, params)
 
-      it 'returns a cached response' do
-        expect(cached_connection.get(endpoint, params)).
-          to eq(CachedSearchApiConnectionResponse.new(response, 'some_cache'))
+        cached_connection.get(endpoint, params)
       end
     end
 
-    context 'on cache miss' do
-      before do
-        allow(cache).to receive(:read).with(endpoint, params).and_return(nil)
-      end
-
-      it 'sends outbound request and cache response' do
-        allow(cached_connection.connection).to receive(:get).with(endpoint, params).and_return(response)
-        allow(cache).to receive(:write).with(endpoint, params, response)
+    context 'when response has been cached' do
+      it 'calls the api' do
+        allow(Rails.cache).to receive(:fetch).and_return(response)
```

Comment:
Can we move this `allow` into a before block to clearly illustrate the context that it is setting and how this context is different from that on ln 34.